packages/wrangler/src/api/pages/deploy.tsx

```diff
@@ -223,11 +225,47 @@ export async function deploy({
 	}
 
 	if (_headers) {
+		const headers = parseHeaders(_headers);
+		const num_valid = headers.rules.length;
+		const num_invalid = headers.invalid.length;
```

Comment:
this is js not python :^)
```suggestion
		const numValid = headers.rules.length;
		const numInvalid = headers.invalid.length;
```